docs/js/modal.md

```diff
@@ -12,16 +12,13 @@ path: modal
 
 ## How to
 
-Modal component allow users to choose which selector to use, you can use a simple `[data-whatever]` or a class. Be sure to not use class like: `.modal`, `.modal-content`, `.modal-body` to initiate modal.
+Modal component allow users to choose which selector to use, you can use a simple `[data-whatever]` or a class. Be sure to not use class like: `.modal`, `.modal-content`, `.modal-body` to initiate modal. Modal has variable `height` based on his content and the `max-height` is `90%`.
```

Comment:
`allows`
`... to not use classes such as: ... , and .modal-body ... a modal.`
Ver a questão sobre alternar no uso do nome do componente e o uso de artigos antes do nome.
Usar `its` ao invés de `the` na última oração.